source/table.cpp

```diff
@@ -41,7 +41,16 @@ spv_context spvContextCreate(spv_target_env env) {
   spvOperandTableGet(&operand_table, env);
   spvExtInstTableGet(&ext_inst_table, env);
 
-  return new spv_context_t{env, opcode_table, operand_table, ext_inst_table};
+  return new spv_context_t{
+      env, opcode_table, operand_table, ext_inst_table,
+      /* A deault message consumer that ignores all messages */
+      [](spvtools::MessageLevel, const char*, const spv_position_t&,
```

Comment:
Now that I know a std::function is comparable to nullptr and assignable from nullptr, then we may as well make the default nullptr since the user could always have set a nullptr function.  When we will invoke the consumer, we always have to check for the nullptr anyways.
Sorry about the flip flop.